programs/ssl/ssl_server2.c

```diff
@@ -4018,7 +4018,8 @@ int main( int argc, char *argv[] )
             ( opt.query_config_mode == DFL_QUERY_CONFIG_MODE ) )
         {
             mbedtls_printf( "Failed to destroy key slot %u - error was %d",
-                            (unsigned) psk_slot, (int) status );
+                            (unsigned) MBEDTLS_SVC_KEY_ID_GET_KEY_ID( psk_slot ),
```

Comment:
Oh nice, I didn't know about this macro. I love learning new things while reviewing PRs :)

Reply:
Didn't know about it before either, but I always like to search for usages of given symbols/types in the IDE I use (eclipse) - this was one of them :)